### ISSUE TITLE
Exceptions are unhandled

### ISSUE THREAD

**Issue by ocean90** [pressjitsu/pj-object-cache-red#20]
PhpRedis can throw a [RedisException](https://github.com/phpredis/phpredis#class-redisexception) in case of connection issues. This may happen in the [get method](https://github.com/pressjitsu/pj-object-cache-red/blob/0c4d8618a0131ad3ad0f74ad24c1dd3f5bb82426/object-cache.php#L582) for example.

I suggest [doing something like redis-cache](https://github.com/tillkruss/redis-cache/blob/a1358acf59e9919b6c54e810015de241ec0d8c31/includes/object-cache.php#L1003-L1009) does. Any thoughts on this?

**Comment by soulseekah** [comment]
Yes, this is a good idea; we should throw exceptions with `WP_DEBUG` on and silence them when off, I think.
